core/src/com/unciv/ui/mapeditor/NewMapScreen.kt

```diff
@@ -49,6 +52,18 @@ class NewMapScreen(val mapParameters: MapParameters = MapParameters()) : PickerS
 
         rightButtonSetEnabled(true)
         rightSideButton.onClick {
+            val message = mapParameters.mapSize.fixUndesiredSizes(mapParameters.worldWrap)
+            if (message != null) {
+                Gdx.app.postRunnable {
```

Comment:
No need for a postRunnable, since we're in an onClick which runs in the main thread anyway, no?

Reply:
Chuckle. I just _knew_ you'd pick up on that. I was too lazy to deep dive into the debugger to verify such an expectation. We're coming from some framework-controlled kind of event, which for me means portions of context are to be treated as a black box. Then we wish to do something that might, just might impact the UI immediately - so make sure you _know_ which thread it'll run on. Plus delaying that portion until the framework is done with whatever else is queued might not be bad... Anyway, that's maybe what I thought. If you're sure...